### DISCUSSION TITLE
能不能把打包后的js从pak文件里挪出来，单独压进一个文件里

### DISCUSSION THREAD

**Discussion by nathan77886** [Tencent/puerts#1283] [Ideas]
这样js代码逻辑的热更就不用依赖ue的pak了

**Comment by chexiongsheng** [comment]
默认的js加载在[DefaultJSModuleLoader.cpp](https://github.com/Tencent/puerts/blob/master/unreal/Puerts/Source/JsEnv/Private/DefaultJSModuleLoader.cpp)

你实现自己的loader，从哪里加载由你自己决定，甚至实时去http服务器下载都行。
